test/e2e/network/netpol/model.go

```diff
@@ -38,11 +48,14 @@ type Model struct {
 	Ports          []int32
 	Protocols      []v1.Protocol
 	DNSDomain      string
+	UDP            bool
```

Comment:
This is already covered by the `Protocols` field -- why not modify that instead?